Resources/misc/tag-steps.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+declare -a commits=(
+	"Create the bundle"
+	"Structure the bundle"
+	"Implement the Tweet\Value class"
+	"Implement the Tweet\Type class"
+	"Register the Field Type as a service"
+	"Implement the Legacy Storage Engine Converter"
+	"Introduce a template"
+	)
+declare -a tags=(
+	"create_the_bundle"
+	"structure_the_bundle"
+	"implement_the_tweet_value_class"
+	"implement_the_tweet_type_class"
+	"register_the_field_type_as_a_service"
+	"implement_the_legacy_storage_engine_converter"
+	"introduce_a_template"
+	)
```

Comment:
I would like to have tags with a numeration.
Like this:
"step1_create_the_bundle"
"step2_structure_the_bundle"
"step3_implement_the_tweet_value_class"
"step4_implement_the_tweet_type_class"
"step5_register_the_field_type_as_a_service"
"step6_implement_the_legacy_storage_engine_converter"
"step7_introduce_a_template